### PR TITLE
Issue 7103 : Added more log lines.

### LINKED ISSUE #7103: Add more logs in EventProcessorCell.
**Is your feature request related to a problem? Please describe.**
Add more logs in EventProcessorCell. It will help to identify the cause of failure if happens in any event processor.

**Describe the solution you'd like**
Add more logs if missing.

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by Bhupender-Y**
**Change log description**  
Add more logs in EventProcessorCell. It will help to identify the cause of failure if happens in any event processor.

**Purpose of the change**  
(Fixes #7103 )

**What the code does**  
This will print more logs in case of failure.

**How to verify it**  
N/A

### CODE REVIEW

File: controller/src/main/java/io/pravega/controller/eventProcessor/impl/EventProcessorCell.java [hunk 1]
```diff
@@ -117,6 +117,7 @@ protected final void run() throws Exception {
                         state.store(event.getPosition());
                     }
                 } catch (Exception e) {
+                    log.debug("Exception while reading the events {}", e);
```
Comment (shshashwat) [inline controller/src/main/java/io/pravega/controller/eventProcessor/impl/EventProcessorCell.java:120]: `Exceptions` should be `warn`

File: controller/src/main/java/io/pravega/controller/eventProcessor/impl/EventProcessorCell.java [hunk 2]
```diff
@@ -168,6 +169,7 @@ private void restart(Throwable error, T event) {
 
         private void handleException(Exception e) {
             ExceptionHandler.Directive directive = eventProcessorConfig.getExceptionHandler().run(e);
+            log.info("Exception handler directive is {}", directive);
```
Comment (shshashwat) [inline controller/src/main/java/io/pravega/controller/eventProcessor/impl/EventProcessorCell.java:172]: here as well, this should be a `warn` log

**Review by shshashwat — CHANGES_REQUESTED** [review]

File: controller/src/main/java/io/pravega/controller/eventProcessor/impl/EventProcessorCell.java (hunk 2 above)
Comment (Bhupender-Y) [reply controller/src/main/java/io/pravega/controller/eventProcessor/impl/EventProcessorCell.java:172]: This log is to print the value of **directive**, so i think having it info level is much better.

File: controller/src/main/java/io/pravega/controller/eventProcessor/impl/EventProcessorCell.java [hunk 3]
```diff
@@ -126,7 +127,7 @@ protected final void run() throws Exception {
 
         @Override
         protected final void shutDown() throws Exception {
-            log.info("Event processor SHUTDOWN {}, state={}", objectId, state());
+            log.info("Event processor SHUTDOWN {}, state={}, ISRUNNING={}", objectId, state(), isRunning());
```
[the remark this answers is not in the document]
Comment (Bhupender-Y) [reply controller/src/main/java/io/pravega/controller/eventProcessor/impl/EventProcessorCell.java:131]: Ideally yes, just to verify that both are same, i updated this.

File: controller/src/main/java/io/pravega/controller/eventProcessor/impl/EventProcessorCell.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (Bhupender-Y) [reply controller/src/main/java/io/pravega/controller/eventProcessor/impl/EventProcessorCell.java:120]: Done

**Review by tkaitchuck — APPROVED** [review]

**Pull request merged**